project_task_project_mandatory/models/project_task.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# © 2015-2017 Elico corp (https://www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    project_id = fields.Many2one(required=True)
```

Comment:
@victormartinelicocorp  Sorry, I misunderstand your point.
About the null value in DB, if there not null value as Eric said, it will quite well; 
but if there exists null value, how to process the record, delete or assign a value at random?
Or when it happend and then handle it, since our database is not null already.